flutter_local_notifications/docs/windows-usage.md

```diff
@@ -0,0 +1,196 @@
+
+<details>
+<summary>Expand to see a full example</summary>
+
+```dart
+final markReadButton = WindowsAction(content: "Mark as Read", arguments: "mark-read");
+
+final deleteButton = WindowsAction(
+  content: "Delete",
+  arguments: "delete-message",
+  buttonStyle: WindowsButtonStyle.critical,
+  tooltip: "Delete this message",
+);
+
+final respondTextInput = WindowsTextInput(
+  id: "reply-message",
+  placeHoldContent: "Enter a message...",
+  title: "Reply",
+);
+
+final presetSelection = WindowsSelection(
+  id: "reply-message",
+  title: "Respond with a preset message",
+  items: [
+    WindowsSelection(id: "reply1", content: "Hello!"),
+    WindowsSelection(id: "reply2", content: "Please leave me alone..."),
+  ],
+);
+
+final details = WindowsNotificationDetails(
+  actions: [markReadButton, deleteButton],
+  inputs: [respondTextInput, presetSelection],
+);
+
+await plugin.show(
+  1, "Alice sent you a message", "Hey! Are you ready?",
+  NotificationDetails(windows: details),
+  payload: "message_ID_123",
+);
+
+void onNotificationTapped(NotificationResponse response) {
+  final messageId = response.payload;
+  switch (response.actionId) {
+    case "delete-message": deleteMessage(messageId);
+    case "mark-read": markRead(messageID, true);
+    case "reply-message":
+      // Inputs on Windows are in NotificationResponse.data
+      final reply = response.data["reply-message"];
+      replyToMessage(messageId, reply);
+  }
+}
+```
+
+</details>
+
+### Custom images and sounds
+
+images can come from different sources: the web, MSIX bundles, files on the user's device, or Flutter assets. See the documentation for [`WindowsImage`](https://pub.dev/documentation/flutter_local_notifications/latest/flutter_local_notifications/WindowsImage-class.html) for complete details on when each time of source is usable.
+
+Sounds can either be one of the [`WindowsNotificationSound`](https://pub.dev/documentation/flutter_local_notifications/latest/flutter_local_notifications/WindowsNotificationSound.html) presets, or a Flutter asset using [`WindowsNotificationAudio.asset()`](https://pub.dev/documentation/flutter_local_notifications/latest/flutter_local_notifications/WindowsNotificationAudio/WindowsNotificationAudio.asset.html). When used in an MSIX package, this constructor will automatically convert the resource to an `ms-appx://` URI, and use a fallback preset sound in debug or non-packaged releases.
+
+### Custom XML
+
+Windows notifications are sent in the form of an XML schema, which the Windows SDK has to parse and verify. Using the `WindowsNotificationDetails` class in this library lets you skip that and build your notification with pure Dart code, but if you wanted to use raw XML for some unsupported or custom behavior, you can:
```

Comment:
- Missing a bit of context towards the end by saying something along the lines of "you can call the following methods"
- The list if missing a colon `:` to separate the methods and description like you have elsewhere